CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Notable changes to the `alacritty_terminal` crate are documented in its
 
 ## 0.16.0-dev
 
+### Changed
+
+-  Shell initialization on macOS to manually check the `~/.hushlogin` file
```

Comment:
This is the user-facing changelog, so we should explain things a little bit. I'd suggest the following:

```suggestion
- Hide login message if `~/.hushlogin` is present
```

This way we're properly documenting the effect it has on the user, rather than some internal details.